content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -365,3 +366,122 @@ HTTP response code, are used in the HTTP response to the client.
 The currently supported reasons are: `Unauthorized`, `Forbidden`, `Invalid`, `RequestEntityTooLarge`.
 If not set, `StatusReasonInvalid` is used in the response to the client.
 
+### Message Expression
```

Comment:
```suggestion
### Message expression
```

To match the "sentience case"  convention for headers in these docs (which I just learned about recently).